libs/wingc/src/json_schema_generator.rs

```diff
@@ -0,0 +1,112 @@
+use crate::{
+	jsify::{codemaker::CodeMaker, JSifier},
+	type_check::{symbol_env::SymbolEnv, Struct, Type, UnsafeRef},
+};
+
+pub(crate) struct JsonSchemaGenerator {}
```

Comment:
If the struct has no fields, I think this is more idiomatic
```suggestion
pub(crate) struct JsonSchemaGenerator;
```